packages/cli/test/unit/openapi/openapi-utils.unit.js

```diff
@@ -41,4 +41,11 @@ describe('openapi utils', () => {
     expect(utils.escapePropertyName('customer_id')).to.eql('customer_id');
     expect(utils.escapePropertyName('customerid')).to.eql('customerid');
   });
+
+  it('escapes chars for comments', () => {
+    expect(utils.escapeComment('/* abc */')).to.eql('\\/* abc *\\/');
```

Comment:
I wonder if we should escape them only if it's a complete comment like `/* .... */` instead of `/* ....` or `... */`. I'm thinking perhaps if it's a malformed/unfinished comment we can fail fast.

Reply:
We should accept what is in the `description` as-is. It's not our responsibility to check the content of OpenAPI spec `description`s.

Reply:
Okay that makes sense.